sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/TPCDSQueryBenchmark.scala

```diff
@@ -58,6 +58,7 @@ object TPCDSQueryBenchmark extends SqlBasedBenchmark with Logging {
       .set("spark.sql.crossJoin.enabled", "true")
       .set("spark.serializer", "org.apache.spark.serializer.KryoSerializer")
       .set("spark.kryo.registrationRequired", "true")
+      .set("spark.sql.ansi.enabled", "false")
```

Comment:
cc @dongjoon-hyun @cloud-fan Now we seem to lose TPCDS compliance by default.

Reply:
Just for my understanding, TPCDS data have some overflows?

Reply:
DIVIDE_BY_ZERO in q90 so far

Reply:
https://issues.apache.org/jira/browse/SPARK-48066 has been filed

Reply:
Thank you!